public/lib/events.js

```diff
@@ -34,7 +34,7 @@ var events = {
 
       // if this was a published event, do an immediate trigger
       var cache = self._cache;
-      if (cache[name]) {
+      if (cache && cache[name]) {
```

Comment:
This is the trace I am looking at: https://bugsnag.com/6wunderkinder-1/webapp/errors/5509d8607c992b99eacff166#comments